### PR TITLE
Fixed a function bind to the instance

### PR DESCRIPTION
**Pull request by zetta83**
Hey @kolodziejczak-sz 👋 
I caught the error `Function called outside component initialization` with context executing function `$set`.
Also, I added yarn things, because it better works instead of npm.

### CODE REVIEW

**Comment by kolodziejczak-sz** [comment]
hey, thank you for the PR, i'll review that today

**Review by kolodziejczak-sz — CHANGES_REQUESTED** [review]
Hi, I really appreciate your work. I've tested your PR and it works. To be honest i wasn't aware that my library doesnt work anymore due to the Svelte repo changes.

I want to merge that fix buuuuuut this PR has way too many unrelated changes;
I don't think the yarn stuff has anything to do with the issue, and i don't consider this change as needed.
the same thing goes with the update of devDependencies.
Don't give me wrong; I think the devDependency update is a great but I would love to see this change in a standalone PR.

To sum up my answer: Could you shrink your pr to the first commit only (dfdc946fa1a4c12bffcd5950127dac99cad1e60d)?

Thanks again for your effort!

**Comment by zetta83** [comment]
Hey, @kolodziejczak-sz !
I created a new [PR](https://github.com/kolodziejczak-sz/svelte-redux-connect/pull/6) with one commit that fixes the error.
